static/app/views/replays/replayTable.tsx

```diff
@@ -16,29 +16,38 @@ import {IconArrow, IconCalendar} from 'sentry/icons';
 import {t} from 'sentry/locale';
 import space from 'sentry/styles/space';
 import type {Organization} from 'sentry/types';
+import {spanOperationRelativeBreakdownRenderer} from 'sentry/utils/discover/fieldRenderers';
 import type {Sort} from 'sentry/utils/discover/fields';
 import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
 import {useLocation} from 'sentry/utils/useLocation';
 import useMedia from 'sentry/utils/useMedia';
 import useOrganization from 'sentry/utils/useOrganization';
 import useProjects from 'sentry/utils/useProjects';
 import {useRoutes} from 'sentry/utils/useRoutes';
-import type {ReplayListLocationQuery, ReplayListRecord} from 'sentry/views/replays/types';
+import type {ReplayItem} from 'sentry/views/performance/transactionSummary/transactionReplays/useReplaysFromTransaction';
+import type {ReplayListLocationQuery} from 'sentry/views/replays/types';
 
 type Props = {
   isFetching: boolean;
-  replays: undefined | ReplayListRecord[];
+  replays: undefined | ReplayItem[];
   showProjectColumn: boolean;
   sort: Sort;
   fetchError?: Error;
+  showSlowestTxColumn?: boolean;
+};
+
+type TableProps = {
+  showProjectColumn: boolean;
+  showSlowestTxColumn: boolean;
 };
 
 type RowProps = {
   minWidthIsSmall: boolean;
   organization: Organization;
   referrer: string;
-  replay: ReplayListRecord;
+  replay: ReplayItem;
```

Comment:
This feels wrong. I'd rather see `ReplayListRecord | ReplayItem`
because when i'm coming from the original relay list page it's odd to see this type that's coming from something very unrelated to the data i've got at hand.